edx_api_doc_tools/conf_utils.py

```diff
@@ -41,7 +41,7 @@ def make_docs_urls(api_info, api_url_patterns=None):
 def make_api_info(
         title="Open edX APIs",
         version="v1",
-        email="oscm@edx.org",
+        email="oscm@openedx.org",
```

Comment:
I don't think this is the right email either -- an Open edX instance running the default configuration shouldn't be directing mail to our OSCM :)

Thing is, I can't think of any other sensible default. How about making this a required argument?

Reply:
A required argument that who will supply? And where?

Reply:
The caller of this function, which is generally a Django project with access to Django settings, would supply it. I believe that's what edx-platform and registrar do currently.

Reply:
In my experience, there's very few emails because of this address (6/year?).  I'm hoping to get this changed so that Axim is the point of contact rather than 2U, without having to change the calling signature.  This function is called without an email argument from a number of places:
```
edx/demographics/demographics/urls.py
edx/edx-exams/edx_exams/urls.py
edx/portal-designer/designer/urls.py
edx/video-encode-manager/encode_manager/urls.py
openedx/blockstore/blockstore/urls.py
openedx/course-discovery/course_discovery/urls.py
openedx/enterprise-access/enterprise_access/urls.py
openedx/enterprise-catalog/enterprise_catalog/urls.py
openedx/license-manager/license_manager/urls.py
```

Changing this argument will be a breaking change, bumping the version number to 2.0.0, which will get upgraded to automatically, and these repos will have to change their calls.  

What would you like to do?

Reply:
Wow, I didn't realize this library was installed into so many services.  I guess we must have put it in the cookiecutter at some point.

`oscm@openedx.org` is good enough. I guess we can change it later if we get too many confused emails.